test/scripts/react_devtools.js

```diff
@@ -26,7 +26,7 @@ Test.describe(`Test React DevTools.`, async () => {
 
   rootComponent.dispatchEvent(new MouseEvent("mouseover", { bubbles: true }));
   await Test.checkHighlighterVisible(true);
-  await Test.checkHighlighterShape("M40,16 L1280,16 L1280,35 L40,35");
+  await Test.checkHighlighterShape("M40,16 L140,16 L140,35 L40,35");
```

Comment:
I might be mistaken here, because I know there are sometimes multiple failures, but if this is the failure that I'm *thinking* of it usually fails without giving any indication that it's a numerical expectation that was failing. Would it be possible for us to move towards a world where failing tests give more of an explicit output like Jest expectations do? Something like:

`expecting 140, got 1280 on line 29 of react_devtools.js` would go a *long* way.